cachelib/s3.py

```diff
@@ -0,0 +1,191 @@
+# -*- coding: utf-8 -*-
+import datetime
+
+try:
+    import cPickle as pickle
+except ImportError:  # pragma: no cover
+    import pickle
+
+try:
```

Comment:
this bit has to go in the `__init__`, else the complete package fails without boto3 installed